styles.css

```diff
@@ -0,0 +1,331 @@
+  overflow: auto;
+}
+
+nav li {
+  float: left;
+}
+
+.leftli {
+  float: right;
+}
+
+nav li a {
+  text-decoration: none;
+  color: blue;
+  display: inline-block;
+  text-align: center;
+}
+
+nav li a.logolink {
+  position: relative;
+  top: 4px;
+  left: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+nav li a img {
+  position: relative;
+  top: 4px;
+  right: 20px;
+}
+
+.headline {
+  background-image: url('image.png');
+  border-bottom-left-radius: 50px;
+  padding: 40px;
+  width: 100%;
+  height: auto;
+  background-color: white;
+}
+
+section .icons {
+  margin-right: 10px;
+}
+
+.textdiv h1 {
+  font-family: 'Poppins', sans-serif;
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.textdiv p {
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+/* Texts headings and Paragraphs Font Poppins Fonts Styling */
+.popheading {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.popp13 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.popp15 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.abouth {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.aboutp {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.abouts {
+  text-decoration: none;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: 2px solid lightgray;
+  border-radius: 5px;
+  display: inline-block;
+  padding: 18px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid gray;
+  border-radius: 8px;
+  margin-top: 40px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 90%;
+
+  /* height: 600px; */
+  background-color: white;
+}
+
+.imgdiv {
+  flex: 100%;
+  margin-left: 20px;
+  margin-top: 20px;
+}
+
+.tonic {
+  flex: 100%;
+  margin-left: 20px;
+  margin-bottom: 20px;
+}
+
+.htcj {
+  background: #ebebff;
+  border-radius: 8px;
+  display: inline-block;
+  padding: 6px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+}
+
+.btn,
+.btn3,
+.btn2,
+.btn4 {
+  border: 2px solid lightgray;
+  border-radius: 5px;
+  display: inline-block;
+  padding: 18px;
+}
```

Comment:
- [ ] I also noticed that your buttons are missing the right hover effects but the `Get My Resume` button in the About section is missing all the effects. 
- [ ] Kindly ensure that you apply these styles while following the Figma design guidelines to improve user interactivity and also so your design can follow the design guidelines in the Figma template :+1: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/213075707-0dcc2cef-b2d5-4fbf-a10f-61f5cf4eb11c.png)

Reply:
Hovering effects added to Get My resume Button. Sending for review again.